homeassistant/components/http.py

```diff
@@ -221,28 +220,6 @@ def send(self, request, filepath):
 _GZIP_FILE_SENDER = GzipFileSender()
 
 
-class HAStaticRoute(StaticRoute):
-    """StaticRoute with support for fingerprinting."""
-
-    def __init__(self, prefix, path):
-        """Initialize a static route with gzip and cache busting support."""
-        super().__init__(None, prefix, path)
-        self._file_sender = _GZIP_FILE_SENDER
-
-    def match(self, path):
-        """Match path to filename."""
-        if not path.startswith(self._prefix):
-            return None
-
-        # Extra sauce to remove fingerprinted resource names
```

Comment:
No, frontend setup is done with all the other setups and thus is doing in the setup phase, which is before the HOMEASSISTANT_START event. However, the frontend will register some urls when that event happens, and we can't control that if it is before/after the HTTP component calls freeze